src/Support/SupportCliCommand.php

```diff
@@ -106,8 +111,8 @@ public function send_diagnostic( /** @noinspection PhpUnusedParameterInspection
 			'is_synthetic' => $is_synthetic,
 		];
 
-		$this->support_data->set_args( $args );
-		$data = $this->support_data->get_data();
+		$support_data = $this->injector->make( SupportData::class, [ 'args' => $args ] );
```

Comment:
Same suggestion as above, creating the `SupportData` object should be simple enough.

```suggestion
		$support_data = new SupportData( $args );
```